sql_metadb/derived_tables/po_line_fund_distribution_transactions.sql

```diff
@@ -0,0 +1,166 @@
+DROP TABLE IF EXISTS folio_derived.po_line_fund_distribution_transactions;
```

Comment:
The leading schema prefix `folio_derived` for derived tables are not necessary anymore and can be deleted anywhere in the query.